src/hmac.rs

```diff
@@ -167,6 +167,7 @@ pub type Signature = Tag;
 pub struct Tag(digest::Digest);
 
 /// A key to use for HMAC signing.
+#[derive(Clone)]
```

Comment:
Similarly, please modify hmac_test.rs to verify that the clone of a `Signing` (resp. `VerificationKey`) gives the same results as the original

Reply:
Done.